docs/UserGuide.md

```diff
@@ -14,6 +14,7 @@
   * [Customize Using csv Config Files](#customize-using-csv-config-files)
     * [`repo-config.csv`](#repo-configcsv)
     * [`author-config.csv`](#author-configcsv)
+    * [`groups-config.csv`](#groups-configcsv)
```

Comment:
Sorry to bring this up now, not sure if plurality is preferred.

@damithc 
Is group~s~-config.csv better?

Reply:
Good catch. In hindsight, all three should have used the plural, to be more accurate of what they contain. 
For the sake of consistency, we can make this `group-config.csv`